### PR TITLE
Add PCA example to ml-integration page [skip ci]

### PR DESCRIPTION
**Pull request by wjxiz1992**
- link implementation repo and use case to ml-integration page.

skip ci since just docs update.

Signed-off-by: Allen Xu <allxu@nvidia.com>

### CODE REVIEW

File: docs/additional-functionality/ml-integration.md [hunk 1]
```diff
@@ -36,3 +36,11 @@ val maxValue = rdd.map(table => {
 You may need to disable RMM caching when exporting data to an ML library as that library
 will likely want to use all of the GPU's memory and if it is not aware of RMM it will not have
 access to any of the memory that RMM is holding.
+
+## Spark ML Algorithms Supported by RAPIDS Accelerator
+
+[PCA(Principal Component Analysis)](https://spark.apache.org/docs/latest/mllib-dimensionality-reduction#principal-component-analysis-pca) has been supported by the plugin directly. (only transform part now)
+
+The implementation leverages the technique of [RapidsUDF](https://github.com/NVIDIA/spark-rapids/blob/branch-21.12/docs/additional-functionality/rapids-udfs.md#rapids-accelerated-user-defined-functions).
```
Comment (gerashegalov) [inline docs/additional-functionality/ml-integration.md:44]:
```suggestion
The plugin implements the `transform` API for [Principal Component Analysis (PCA)](https://spark.apache.org/docs/latest/mllib-dimensionality-reduction#principal-component-analysis-pca) as an [accelerated UDF](https://github.com/NVIDIA/spark-rapids/blob/branch-21.12/docs/additional-functionality/rapids-udfs.md#rapids-accelerated-user-defined-functions).
```

File: docs/additional-functionality/ml-integration.md [hunk 2]
```diff
@@ -36,3 +36,11 @@ val maxValue = rdd.map(table => {
...
+
+You may find the details in [spark-rapids-ml](https://github.com/NVIDIA/spark-rapids-ml) and the use case in [spark-rapids-example's pca subfolder](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-21.10/examples/pca).
```
Comment (gerashegalov) [inline docs/additional-functionality/ml-integration.md:46]:
```suggestion
You can find the details in [spark-rapids-ml](https://github.com/NVIDIA/spark-rapids-ml) and the use case in [spark-rapids-example's pca subfolder](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-21.10/examples/pca).
```

**Review by gerashegalov — DISMISSED** [review]

File: docs/additional-functionality/ml-integration.md [hunk 3]
```diff
@@ -36,3 +36,9 @@ val maxValue = rdd.map(table => {
 You may need to disable RMM caching when exporting data to an ML library as that library
 will likely want to use all of the GPU's memory and if it is not aware of RMM it will not have
 access to any of the memory that RMM is holding.
+
+## Spark ML Algorithms Supported by RAPIDS Accelerator
+
+The plugin implements the `transform` API for [Principal Component Analysis (PCA)](https://spark.apache.org/docs/latest/mllib-dimensionality-reduction#principal-component-analysis-pca) as an [accelerated UDF](https://github.com/NVIDIA/spark-rapids/blob/branch-21.12/docs/additional-functionality/rapids-udfs.md#rapids-accelerated-user-defined-functions).
+
+You can find the details in [spark-rapids-ml](https://github.com/NVIDIA/spark-rapids-ml) and the use case in [spark-rapids-example's pca subfolder](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-21.10/examples/pca).
```
Comment (jlowe) [inline docs/additional-functionality/ml-integration.md:44]: Lines should be wrapped at approx 100 characters to make viewing in a terminal and identifying changes per line easier.

Also the `transform` API is _not_ implemented by the RAPIDS Accelerator, it's done as a separate example.  All the RAPIDS Accelerator does here is provide the RAPIDS accelerated UDF framework.  All the stuff specific to PCA is not provided directly.

Suggest something more like:
```suggestion
The [spark-rapids-examples repository](https://github.com/NVIDIA/spark-rapids-examples) provides a
[working example](https://github.com/NVIDIA/spark-rapids-examples/tree/branch-21.10/examples/pca)
of accelerating the `transform` API for
[Principal Component Analysis (PCA)](https://spark.apache.org/docs/latest/mllib-dimensionality-reduction#principal-component-analysis-pca).
The example leverages the [RAPIDS accelerated UDF interface](rapids-udfs.md) to provide a native
implementation of the algorithm. The details of the UDF implementation can be found in the
[spark-rapids-ml repository](https://github.com/NVIDIA/spark-rapids-ml).
```

**Review by jlowe — APPROVED** [review]

**Review by gerashegalov — APPROVED** [review]
LGTM

**Comment by firestarman** [comment]
build

**Review by sameerz — APPROVED** [review]

**Comment by wjxiz1992** [comment]
build

**Pull request merged**